Microsoft.HealthVault.Fhir/Microsoft.HealthVault.Fhir.csproj

```diff
@@ -6,9 +6,21 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="Data\UnitConversions.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="Data\UnitConversions.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
```

Comment:
Talking to @gcorvera, we're going to leave this as a content file for now to allow end users to potentially extend.